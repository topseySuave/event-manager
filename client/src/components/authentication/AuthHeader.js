import React, { Component } from 'react'
import { Link } from 'react-router-dom'

class AuthHeader extends Component {
    render(){
        return (
            <nav className="wow fadeInDown black-text gradient__bg">
                <div className="container">
                    <div className="nav-wrapper">
                        <Link to="/" className="brand-logo center logo">
                            <p>Boots EM</p>
                        </Link>
                    </div>
                </div>
            </nav>
        );
    }
}

export default AuthHeader;