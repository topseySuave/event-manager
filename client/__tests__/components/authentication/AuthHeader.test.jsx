import React from 'react';
import { shallow } from 'enzyme';
import AuthHeader from
  '../../../src/components/authentication/AuthHeader';

describe('Authentication Header Component', () => {
  const wrapper = shallow(<AuthHeader />);
  test('make a snapshot of the AuthHeader component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
