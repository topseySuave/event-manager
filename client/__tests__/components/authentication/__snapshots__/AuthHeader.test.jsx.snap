// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication Header Component make a snapshot of the AuthHeader component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AuthHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        <div
          className="nav-wrapper"
        >
          <Link
            className="brand-logo center logo"
            href="/"
            replace={false}
            to="/"
          >
            <p>
              Boots EM
            </p>
          </Link>
        </div>
      </div>,
      "className": "wow fadeInDown black-text gradient__bg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="nav-wrapper"
        >
          <Link
            className="brand-logo center logo"
            href="/"
            replace={false}
            to="/"
          >
            <p>
              Boots EM
            </p>
          </Link>
        </div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            className="brand-logo center logo"
            href="/"
            replace={false}
            to="/"
          >
            <p>
              Boots EM
            </p>
          </Link>,
          "className": "nav-wrapper",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <p>
              Boots EM
            </p>,
            "className": "brand-logo center logo",
            "href": "/",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Boots EM",
            },
            "ref": null,
            "rendered": "Boots EM",
            "type": "p",
          },
          "type": [Function],
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          <div
            className="nav-wrapper"
          >
            <Link
              className="brand-logo center logo"
              href="/"
              replace={false}
              to="/"
            >
              <p>
                Boots EM
              </p>
            </Link>
          </div>
        </div>,
        "className": "wow fadeInDown black-text gradient__bg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="nav-wrapper"
          >
            <Link
              className="brand-logo center logo"
              href="/"
              replace={false}
              to="/"
            >
              <p>
                Boots EM
              </p>
            </Link>
          </div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="brand-logo center logo"
              href="/"
              replace={false}
              to="/"
            >
              <p>
                Boots EM
              </p>
            </Link>,
            "className": "nav-wrapper",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <p>
                Boots EM
              </p>,
              "className": "brand-logo center logo",
              "href": "/",
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Boots EM",
              },
              "ref": null,
              "rendered": "Boots EM",
              "type": "p",
            },
            "type": [Function],
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
